src/core/RemotePlugin.cpp

```diff
@@ -222,6 +222,7 @@ bool RemotePlugin::init(const QString &pluginExecutable,
 	// FIXME what should I pass as the ID?
 	args << QString::number(Engine::getSong()->vstSyncController().sharedMemoryKey());
 	args << extraArgs;
+	args << QCoreApplication::applicationPid();
```

Comment:
빌드 오류가 나는 걸 뒤늦게 확인했습니다.
```suggestion
	args << QString::number(QCoreApplication::applicationPid());
```

아예 revert 후 upstream에 최근에 추가된 `RemoteVstPlugin` 자동 종료 코드를 가져오는 게 나을 수도 있을 것 같습니다(Windows 코드 한정).